pkg/services/cosmosdb/common-update.go

```diff
@@ -15,3 +17,66 @@ func (
 ) GetUpdater(service.Plan) (service.Updater, error) {
 	return service.NewUpdater()
 }
+
+// This function will build a map in which only read regions changed.
+func (c *cosmosAccountManager) buildGoTemplateParamsOnlyRegionChanged(
+	provisioningParameters *service.ProvisioningParameters,
+	updatingParameters *service.ProvisioningParameters,
+	dt *cosmosdbInstanceDetails,
+	kind string,
+) (map[string]interface{}, error) {
+	p := map[string]interface{}{}
+	p["name"] = dt.DatabaseAccountName
+	p["kind"] = kind
+	p["location"] = provisioningParameters.GetString("location")
+	p["readRegions"] = updatingParameters.GetStringArray("readLocations")
+	if provisioningParameters.GetString("autoFailoverEnabled") == "enabled" {
```

Comment:
better to ignore case?
and also define a const string enabled = "enabled" just like the disabled.

Reply:
Thanks, will change the code